bigchaindb/common/exceptions.py

```diff
@@ -7,85 +7,98 @@ class ConfigurationError(BigchainDBError):
     """Raised when there is a problem with server configuration"""
 
 
-class OperationError(BigchainDBError):
-    """Raised when an operation cannot go through"""
-
-
-class TransactionDoesNotExist(BigchainDBError):
-    """Raised if the transaction is not in the database"""
+class DatabaseAlreadyExists(BigchainDBError):
```

Comment:
Yes but thats not the scope of this PR. It's just highlighted because the order was switched around.